bundler/lib/bundler/cli/gem.rb

```diff
@@ -145,6 +145,12 @@ def run
         config[:ci_config_path] = ".circleci "
       end
 
+      if ask_and_set(:zeitwerk, "Do you want to use Zeitwerk to load classes?",
+        "With Zeitwerk (https://github.com/fxn/zeitwerk), Ruby can load classes automatically " \
+          "based on name conventions so that you don't have to require files manually.")
+        config[:zeitwerk] = true
+      end
```

Comment:
Should we ask for this, which would set a `BUNDLE_GEM__ZEITWERK` config setting that will be remembered across `bundle gem` calls?

While one may want zeitwerk sometimes (or often), I don't know if it is a setting that should be remembered as if you are doing a very simple gem with like 2-3 files, one might want to forego zeitwerk to keep their gem lean and avoid a third-party dependency. But because they could have enabled zeitwerk before, zeitwerk would always be enabled. I guess in this case one could always provide `--no-zeitwerk`, so maybe I am splitting hairs here.

Reply:
@jeromedalbert  I went with the same approach it uses for the CoC or the MIT license, which results in question + `--zeitwerk/--no-zeitwerk` option + memorized answer in `~/.bundle/config`. I personally think it's fine and consistent but happy to change it if you want me to.